src/control/cmd/dmg/pool_test.go

```diff
@@ -1012,6 +1012,32 @@ func TestPoolCommands(t *testing.T) {
 			}, " "),
 			nil,
 		},
+		{
+			"Query pool with UUID, enabled ranks and disabled ranks",
+			"pool query --show-disabled --show-enabled 12345678-1234-1234-1234-1234567890ab",
+			strings.Join([]string{
+				printRequest(t, &control.PoolQueryReq{
+					ID: "12345678-1234-1234-1234-1234567890ab",
+					QueryMask: setQueryMask(func(qm *daos.PoolQueryMask) {
+						qm.SetOptions(daos.PoolQueryOptionEnabledEngines, daos.PoolQueryOptionDisabledEngines)
+					}),
+				}),
+			}, " "),
+			nil,
+		},
+		{
+			"Query pool with UUID, enabled ranks and disabled ranks",
+			"pool query -b -e 12345678-1234-1234-1234-1234567890ab",
+			strings.Join([]string{
+				printRequest(t, &control.PoolQueryReq{
+					ID: "12345678-1234-1234-1234-1234567890ab",
+					QueryMask: setQueryMask(func(qm *daos.PoolQueryMask) {
+						qm.SetOptions(daos.PoolQueryOptionEnabledEngines, daos.PoolQueryOptionDisabledEngines)
+					}),
+				}),
+			}, " "),
+			nil,
+		},
```

Comment:
IMO it's not necessary to duplicate it with the short versions of params, as it doesn't test anything different in our code.

Reply:
...but it does ensure consistency with commands that expect both long and short versions of the option. i.e. verification that the short versions are supported

Reply:
@tanabarr indicated this test was to check that the behavior of long and short options are consistent.

Reply:
Fair point, let's leave it then.